setup.cfg

```diff
@@ -1,7 +1,52 @@
-[versioneer]
-VCS = git
-style = pep440-branch-based
-versionfile_source = nc_time_axis/_version.py
-versionfile_build = nc_time_axis/_version.py
-tag_prefix = v
-parentdir_prefix = nc-time-axis-
+[metadata]
+author = SciTools Developers
+classifiers =
+    Development Status :: 2 - Pre-Alpha
```

Comment:
Are we still happy that this is true?

Reply:
Note sure, see https://pypi.org/classifiers/ for PyPI classifier choices.

Happy to go with `Development Status :: 5 - Production/Stable`... happy to go with whatever you believe is best :+1:

Reply:
@lbdreyer I've changed this to be the above, does that seem reasonable?